charts/k8s-monitoring-v1/templates/alloy_config/_processors.alloy.txt

```diff
@@ -173,6 +174,7 @@ otelcol.processor.transform "default" {
       "set(attributes[\"namespace\"], attributes[\"k8s.namespace.name\"])",
       "set(attributes[\"loki.resource.labels\"], \"{{ .Values.logs.receiver.transforms.labels | join ", " }}\")",
```

Comment:
No longer relevant if we send OTLP logs to the Grafana Cloud OTLP Endpoint and adopt the new Loki format for OTLP logs.
`loki.resource.labels` will be ignored or, worse, added as a meaningless resource attribute ie a metadata.
This comment applies to all the files where this config block is added.